administrator/components/com_menus/src/Model/MenuModel.php

```diff
@@ -364,7 +364,7 @@ public function &getModules()
         foreach ($modules as &$module) {
             $params = new Registry($module->params);
 
-            $menuType = $params->get('menutype');
+            $menuType = (string) $params->get('menutype');
```

Comment:
```suggestion
            $menuType = $params->get('menutype', '');
```

Would be better, I think.